redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1219 @@
+            if not mutes_data:
+                to_del.append(channel_id)
+                continue
+            if channel_id in [c.id for c in ctx.guild.channels]:
+                msg += _("__<#{channel_id}> Mutes__\n").format(channel_id=channel_id)
+                for user_id, mutes in mutes_data.items():
+                    if not mutes:
+                        continue
+                    user = ctx.guild.get_member(user_id)
+                    if not user:
+                        user_str = f"<@!{user_id}>"
+                    else:
+                        user_str = user.mention
+                    if mutes["until"]:
+                        time_left = timedelta(
+                            seconds=mutes["until"] - datetime.now(timezone.utc).timestamp()
+                        )
+                        time_str = humanize_timedelta(timedelta=time_left)
+                    else:
+                        time_str = ""
+                    msg += _("{member} ").format(member=user_str)
+                    if time_str:
+                        msg += _("__Remaining__: {time_left}\n").format(time_left=time_str)
+                    else:
+                        msg += "\n"
+        for c in to_del:
+            del self._channel_mutes[c]
+        if msg:
+            for page in pagify(msg):
+                await ctx.maybe_send_embed(page)
+            return
+        await ctx.maybe_send_embed(_("There are no mutes on this server right now."))
+
+    @commands.command(usage="[users...] [time_and_reason]")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_roles=True)
+    async def mute(
+        self,
+        ctx: commands.Context,
+        users: commands.Greedy[discord.Member],
+        *,
+        time_and_reason: MuteTime = {},
+    ):
+        """Mute users.
+
+        `[users...]` is a space separated list of usernames, ID's, or mentions.
+        `[time_and_reason]` is the time to mute for and reason. Time is
+        any valid time length such as `30 minutes` or `2 days`. If nothing
+        is provided the mute will be indefinite.
+
+        Examples:
+        `[p]mute @member1 @member2 spam 5 hours`
+        `[p]mute @member1 3 days`
+
+        """
+        if not users:
+            return await ctx.send_help()
+        if ctx.me in users:
+            return await ctx.send(_("You cannot mute me."))
+        if ctx.author in users:
+            return await ctx.send(_("You cannot mute yourself."))
+        if not await self._check_for_mute_role(ctx):
+            return
+        async with ctx.typing():
+            duration = time_and_reason.get("duration", {})
+            reason = time_and_reason.get("reason", None)
+            time = ""
+            until = None
+            if duration:
+                until = datetime.now(timezone.utc) + timedelta(**duration)
+                time = _(" for ") + humanize_timedelta(timedelta=timedelta(**duration))
```

Comment:
With this amount of context, it might be hard for translator to translate this properly, maybe something like this would be better? (I'm not sure)
```suggestion
                duration_str = humanize_timedelta(timedelta=timedelta(**duration))
                time = _(" for {duration}").format(duration=duration_str)
```